python/tvm/relax/frontend/nn/core.py

```diff
@@ -540,6 +540,56 @@ def _compile(spec, device, pipeline, debug):
         raise ValueError(f"Unknown out_format: {out_format}")
 
 
+class ModuleDict(Module):
+    """Holds submodules in a dict."""
```

Comment:
@mshr-h 

*nit*
Just a tiny little bit more words here about what "submodules" are. 
Noticed that all classes here have a bit more explanative comment on them.